### ISSUE TITLE
placeholders in a row not working in config and locale

### ISSUE THREAD

**Issue by Feeriix** [Zrips/CMI#3290]
**Description of issue:**

With the recent update of not translating placeholders with double %, adding placeholders in a row in config or locale shows just text and does not translate the placeholder.
---

[Image](https://user-images.githubusercontent.com/6758392/81532210-5167ef00-9364-11ea-873e-8fc84cdae888.png)

Example:
```
afkSymbolOff: '%luckperms_suffix_element_highest_on_track_tier1%%luckperms_prefix_element_highest_on_track_tier1%%luckperms_prefix_element_highest_on_track_donor%%luckperms_prefix_element_highest_on_track_staff%'
```
Adding spaces between placeholders work as intended. 

**Cmi Version (using`/cmi version`):** 8.6.24.3

**Server Type (Spigot/Paperspigot/etc):** Paperspigot

**Server Version (using `/ver`):** git-Paper-260 (MC: 1.15.2)

**Relevant plugins (Delete if this isn't needed):** PlaceholderAPI, LuckPerms

**Comment by Zrips** [comment]
Good catch, will fix this one with next build.

**Comment by Feeriix** [comment]
@Zrips still not fixed in v8.6.25.1
